src/retrace/retrace.py

```diff
@@ -1125,13 +1125,13 @@ def prepare_debuginfo(self, task, chroot=None, kernelver=None):
                 kernel_path = kernel_path + "."
             kernel_path = kernel_path + str(kernelver.flavour)
 
-        vmlinux_cache_path = debugdir_base + "usr/lib/debug/lib/modules" / kernel_path / "vmlinux"
+        vmlinux_cache_path = debugdir_base / "usr/lib/debug/lib/modules" / kernel_path / "vmlinux"
         if vmlinux_cache_path.is_file():
-            log_info("Found cached vmlinux at path: " + vmlinux_cache_path)
+            log_info("Found cached vmlinux at path: " + str(vmlinux_cache_path))
```

Comment:
I would slightly prefer:

`"Found cached vmlinux at path: {}".format(vmlinux_cache_path)

Many people find it more readable.
`

Reply:
Thanks! It's actually safer because `.format()` works even if the arg is a `Path`, unlike `+`.